<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="search">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>120</y>
      <width>411</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  background-color: rgba(138, 79, 38, 65%);
   border:2px solid #cacaca;
border-radius:10px;
color:white;
font: 11pt &quot;楷体&quot; Black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="passward">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  background-color: rgba(138, 79, 38, 65%);
   border:2px solid #cacaca;
border-radius:10px;
color:white;
font: 11pt &quot;楷体&quot; Black;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>440</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>登  陆</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="stuRadioButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>115</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>学生</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="managerRadioButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>管理员</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
